### PR TITLE
Related content

### PR DESCRIPTION
**Pull request by kjell**
- [x] make it look nice
- [x] show audio in an `<audio>` element for quick playing

### CODE REVIEW

**Comment by emhavens** [comment]
Here is a possible design for the related content. The background images for the link will populate with images taken from the related content. So the overlay will remain the same for art stories. For News Flash it would reflect the title and an image from the News Flash would be the background. The icon is a place holder until we get our full set from Pentagram.

I am going to start building it out so we can see how it functions in reality. If you have other ideas or suggestions for this one don't hesitate to share!

related-content

**Comment by emhavens** [comment]
I have made the related content changes above. The biggest things missing are the swapping out images per object and content. All of them have a static background image. 

I also styled the action links bar (like, email, download) Waiting on share and any other icons that might be useful.

If you want to view the changes or alter them be sure to get latest on related-content branch and you should be set to view those changes.

File: sass/main.scss [hunk 1]
```diff
@@ -622,8 +657,56 @@ h5.details-title {
...
+.audioClip a {
+  top: 13px;
+}
+.newsflash, .artstory, .explore-content {
+  display: inline-block;
+  width: 175px;
+  height: 175px;
+  float: left;
+  margin:20px 10px 0 0;
+  background-image: url("http://api.artsmia.org/images/1218/400/medium.jpg");
```
Comment (kjell) [inline sass/main.scss]: if this goes in as an inline style we can use the correct ID

**Comment by kjell** [comment]
Deployed the newest to staging. 

- [ ] We'll need to take a look at the white text over light images: http://collection.staging.artsmia.org/art/529/lucretia-rembrandt-harmensz-van-rijn

image

- [ ] the boxes don't line up right on [Lucretia](http://collection.staging.artsmia.org/art/529/lucretia-rembrandt-harmensz-van-rijn), but they do on [Olive Trees](http://collection.staging.artsmia.org/art/1218)?

**Pull request merged**